migrations/server/mysql/0004_drop_timestamp_key.up.sql

```diff
@@ -0,0 +1 @@
+DROP TABLE `timestamp_keys`;
```

Comment:
Yeah I'm not sure either.  I guess you really only run migrations once, so creating and then destroying a table isn't an issue.